Kernel/System/ProcessManagement/TransitionAction/TicketStateSet.pm

```diff
@@ -83,6 +83,16 @@ sub Params {
             Value    => '1 (can overwrite the logged in user)',
             Optional => 1,
         },
+        {
+            Key      => 'PendingTime',
```

Comment:
PendingTimeDiff is a valid parameter, but PendingTime doesn't seem to be one. Could you please quickly confirm, that it works in the standard?

Reply:
PendingTime and PendingTimeDiff seem to be used and documented for TicketCreate, so I would consider both feasible. But I will have them tested.